assets/_sass/_default-variables.scss

```diff
@@ -15,7 +15,7 @@ $color-yellow-green: #beb239;
 $color-green: #84a93f;
 $color-blue: #4586cd;
 $color-red: #e52213;
-
+$color-black: #3d3d3d;
```

Comment:
Small nitpick on naming, but on enterprise, we have this colour as `veryDarkGray`. Black should be #000

Reply:
FYI the variables especially the color names are a mess. Plan to do some cleanup there next friday